apps/designer/app/shared/shortcuts/index.ts

```diff
@@ -5,6 +5,7 @@ export const shortcuts = {
   preview: "cmd+shift+p, ctrl+shift+p",
   copy: "cmd+c, ctrl+c",
   paste: "cmd+v, ctrl+v",
+  delete: "backspace, delete",
```

Comment:
This is a runtime change and perhaps this should be reverted. I've added this to satisfy this call:
https://github.com/webstudio-is/webstudio-designer/blob/b913ac73d726ad41ca33bda2d44f1bd6fdf09ea9/apps/designer/app/designer/features/topbar/menu/menu.tsx#L192-L195
and I've matched the keys declared here:
https://github.com/webstudio-is/webstudio-designer/blob/b913ac73d726ad41ca33bda2d44f1bd6fdf09ea9/apps/designer/app/canvas/shared/use-shortcuts.ts#L107

Reply:
why would it need a revert? seems like a legit fix, I will go through all shortcuts in the end and test them